mods/ra/maps/allies-05a/rules.yaml

```diff
@@ -115,7 +115,7 @@ Colt:
 		Image: AGUN
 	Armament:
 		Weapon: MissionColt
-		LocalOffset: 432,150,-30, 432,-150,-30
+		LocalOffset: 520,100,450, 520,-150,450
```

Comment:
A better approach would be to remove this hacktor, put the dummy weapon on the prison, and then script it to attack itself.